libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -247,13 +254,26 @@ data Api routes = Api
         :> "otr"
         :> "messages"
         :> ReqBody '[Servant.JSON, Proto] Public.NewOtrMessage
+        :> UVerb 'POST '[Servant.JSON] PostOtrResponsesUnqualified,
+    postOtrMessage ::
+      routes
+        :- Summary "Post an encrypted message to a conversation (accepts JSON or Protobuf)"
```

Comment:
The description looks wrong, since this endpoint does not accept JSON.

(observed by @mdimjasevic)